### PR TITLE
blog maintenance…

### PR DESCRIPTION
**Pull request by designfirst**
fix styles, add more documentation, encrypt slack token, remove duplicate title from post

### CODE REVIEW

File: _posts/2018-03-05-airing-out-a-new-job-system.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Airing out a new job system"
+title: "Airing Out A New Job System"
```
Comment (tmccartan) [inline _posts/2018-03-05-airing-out-a-new-job-system.md]: should the O be upper case here?

**Comment by tmccartan** [comment]
Probably a bad change but I think if you change &_body to have text-align:justify it looks a bit nice also. Just a suggestion

File: _posts/2018-03-05-airing-out-a-new-job-system.md (hunk 1 above)
Comment (designfirst) [reply _posts/2018-03-05-airing-out-a-new-job-system.md]: yes ;)

**Comment by designfirst** [comment]
@tmccartan feel free to push a commit to fix typos!

**Comment by tmccartan** [comment]
@designfirst good to go from me, thanks for the maintenance

**Pull request merged**